src/primitives/block.h

```diff
@@ -25,7 +39,7 @@ class CBlockHeader
 public:
     static const int32_t CURRENT_VERSION=SIZE_FORK_VERSION;
     // This code knows the rules for these block versions:
-    static const int32_t UNDERSTOOD_VERSIONS = (SIZE_FORK_VERSION | 3 | 2 | 1);
+    static const int32_t UNDERSTOOD_VERSIONS = (SIZE_FORK_VERSION | 4 | 3 | 2 | 1);
```

Comment:
Block version 3 (BIP66) used version 3.

It's worth mentioning that the usage of 3 was in HEAD; I just added 4 because it seems to have been overlooked when BIP65 support was added in 0.11.2 to BitPay's BIP101 branch.